### DISCUSSION TITLE
How to use multi-caret editing feature?

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/ecode#304] [Q&A] [answered]
Is there a tutorial for the editor? I don't see anything other than the README.md of this repository. I have never used lite or lite-xl. This editor is completely new to me.

**Comment by SpartanJ** [comment, marked as answer]
Having a selection you can press Ctrl+D to create a new cursor over the next match, repeat it as many times as wanted. Ctrl+Shift+D will create a new cursor for every existing match. You can also add a new cursor by simply Ctrl+Click over any editor position. That's basically it.

**Reply by SpartanJ** [reply]
[Here](https://github.com/SpartanJ/ecode/releases/tag/ecode-0.4.3) there's more information and a short video.

**Reply by SpartanJ** [reply]
BTW: the editor is not based on lite-xl, it just supports some of its formats but it's a completely different application. I should make it clearer in the README.
